app/helpers/Pagination.php

```diff
@@ -83,4 +111,50 @@ public function getOriginalOrderBy(): ?string {
     return $this->originalOrderBy;
   }
 
+  /**
+   * True if filter of given name is present.
+   */
+  public function hasFilter(string $name)
+  {
+    return array_key_exists($name, $this->filters);
+  }
+
+  /**
+   * Return the filter value.
+   * @throws InternalServerErrorException If the filter is not present.
+   */
+  public function getFilter(string $name)
+  {
+    if (!$this->hasFilter($name)) {
+      throw new InternalServerErrorException("Filter $name is not present.");
+    }
+    return $this->filters[$name];
+  }
+
+  public function getRawFilters()
+  {
+    return $this->filters;
+  }
+
+
+  public function getLocale()
+  {
+    return $this->locale;
+  }
+
+  /**
+   * Find appropriate MySQL Collation for given locale.
+   */
+  public function getLocaleCollation()
```

Comment:
I am not happy about it either. But there are very few options. Only other two choices would be to embed it in OrderByCollationInjectionMysqlWalker (which is slightly worse in my opinion) or place it in a separate helper (which is overkill in case of such a tiny function). If you have other alternatives in mind which I have overlooked, feel free to suggest them... :)

Reply:
For me, the better place would be the `OrderByCollationInjectionMysqlWalker`, but I do not insist on it, if you think it would be worse...